### PR TITLE
SQMAGOPC-253 Brand changes

### PR DESCRIPTION
**Pull request by richard-madarasz**

### CODE REVIEW

File: view/frontend/web/template/payment/checkout-bypass.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
             <div class="checkout-logo">
                 <div class="checkout-logo-container">
                     <img src="https://cdn2.hubspot.net/hubfs/2610868/Logot/op-logo-30x30px.png"
```
Comment (loueranta) [inline view/frontend/web/template/payment/checkout-bypass.html]: I just remembered that we have to update the logo as well! 
I'll upload new logo to HS CDN and update the link before we merge this

File: view/frontend/web/template/payment/checkout.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
             <span data-bind="text: getTitle()"></span>
             <div class="checkout-logo">
                 <div class="checkout-logo-container">
-                    <img src="https://cdn2.hubspot.net/hubfs/2610868/Logot/op-logo-30x30px.png" alt="OP Payment Service">
```
Comment (loueranta) [inline view/frontend/web/template/payment/checkout.html:12]: Another reference to a logo we have to update.

**Review by loueranta — APPROVED** [review]

**Review by loueranta — APPROVED** [review]

File: Setup/Recurring.php [hunk 3]
```diff
@@ -30,7 +30,7 @@ class Recurring implements InstallSchemaInterface
     /**
      * @var string
      */
-    const ORDER_STATUS_CUSTOM_LABEL = 'Pending OP Payment Service';
+    const ORDER_STATUS_CUSTOM_LABEL = 'Pending Checkout Finland';
```
Comment (Seppo-Konttinen-Solteq) [inline Setup/Recurring.php:33]: Does this handle renaming of already added custom order statuses or do we need script to rename the already existing and is there change that something brakes if there are open orders with the old status codes?

File: i18n/fi_FI.csv [hunk 4]
```diff
@@ -68,21 +68,21 @@ Shipping,Toimitus
 "Please navigate to your Magento orders, select the order in question, and click restore order. Note the stock.", "Ole hyvä, ja valitse kyseinen tilaus Magenton hallinnassa ja klikkaa Palauta tilaus. Huomioi varastosaldo."
 "Restore Order", "Palauta tilaus"
 "Are you sure you want to make changes to this order?", "Haluatko varmasti tehdä muutoksia tähän tilaukseen?"
-"Pending payment from OP Payment Service","Odottaa maksua OP Payment Servicestä"
-"Order canceled. Failed to redirect to OP Payment Service.","Tilaus peruutettu. Ohjausta OP Payment Serviceen ei voitu tehdä"
-"Op Payment Service API credentials are incorrect. Please contact support.","Op Payment Service tunnuksia ei löytynyt. Ole hyvä, ja ota yhteyttä asiakaspalveluun."
+"Pending payment from Checkout Finland","Odottaa maksua Checkout Finlandistä"
```
Comment (Seppo-Konttinen-Solteq) [inline i18n/fi_FI.csv:71]: Odottaa maksua Checkout Finlandista

Is this correct @loueranta

**Review by Seppo-Konttinen-Solteq — CHANGES_REQUESTED** [review]

File: Setup/Recurring.php (hunk 3 above)
Comment (richard-madarasz) [reply Setup/Recurring.php:33]: It only sets this status for newly created orders. It should not break old codes as there is no reference to it anywhere in the code

Comment (Seppo-Konttinen-Solteq) [reply Setup/Recurring.php:33]: @richard-madarasz ok, we can do some db updating later on when we implement code changes.

File: i18n/fi_FI.csv (hunk 4 above)
Comment (Seppo-Konttinen-Solteq) [reply i18n/fi_FI.csv:71]: @richard-madarasz could you still change:

"Odottaa maksua Checkout Finlandistä" to "Odottaa maksua Checkout Finlandista"

**Comment by Seppo-Konttinen-Solteq** [comment]
@richard-madarasz if you can do that small text change, no need for re-reviewing. Then deploy QA_V1.3 to test environment. That is done via Jenkins, use m241os version. If you need help please ask. Then it can be tested here https://m241os.opcheckout.magentokehitys.fi/test

**Review by Seppo-Konttinen-Solteq — APPROVED** [review]

**Pull request merged**